crates/utils/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod env_utils;
```

Comment:
Do we need a full crate for this? cc @notlesh

Reply:
@heemankv I would suggest so. Or just inline the code and skip the functions part.

Reply:
Alright added, can you re-review @whichqua